modules/concepts/hooks/use-hooks-on-modern-pages.md

```diff
@@ -154,6 +154,10 @@ Prestashop automatically checks if modules have a `config/services.yml` file and
 ./bin/console cache:clear --no-warmup
 ```
 
+In case Prestashop failed to autoload automatically, [you can generate the autoload files][setup-composer] with composer manually.
```

Comment:
```suggestion
If PrestaShop fails to load files automatically, [you can generate the autoloader][setup-composer] with Composer. This will create a `vendor` directory with a Composer-based autoloader inside.
```